README.md

```diff
@@ -52,11 +52,27 @@ The package helps collect device data for merchants with direct integration (sta
         val yourConfig = RiskConfig(publicKey = "pk_qa_xxx", environment = RiskEnvironment.QA)
 
         try {
-            val riskInstance = Risk.getInstance(yourConfig)
-            riskInstance?.let {
-            val response = it.publishData()
-            println(response)
-            }
+            val riskInstance =
+                Risk.getInstance(
+                    context,
+                    RiskConfig(
+                        BuildConfig.SAMPLE_MERCHANT_PUBLIC_KEY,
+                        RiskEnvironment.QA,
+                        false,
+                    ),
+                ).let {
+                    it?.let {
+                        it
+                    } ?: run {
+                        null
+                    }
+                }
+
+          riskInstance?.publishData().let {
+              if (it is PublishDataResult.Success) {
+                  println("Device session ID: ${it.deviceSessionId}")
+              }
+          }
```

Comment:
This feels a bit incomplete - I think this should be separated because we want them to instantiate Risk early on but only publish data later when submitting the payment